src/EnergyPlus/HVACManager.cc

```diff
@@ -3005,6 +3009,30 @@ namespace HVACManager {
         }
     }
 
+    void CheckAirLoopFlowBalance()
+    {
+        // Check for unbalanced airloop
+        if (!isPulseZoneSizing && !DataHeatBalance::ZoneAirMassFlow.EnforceZoneMassBalance && !DataGlobals::WarmupFlag && AirLoopsSimOnce) {
+            for (int AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum) {
+                auto &thisAirLoopFlow(AirLoopFlow(AirLoopNum));
+                if (!thisAirLoopFlow.FlowError) {
+                    Real64 unbalancedExhaustDelta = thisAirLoopFlow.SupFlow - thisAirLoopFlow.OAFlow - thisAirLoopFlow.SysRetFlow;
```

Comment:
Also out of your scope, but this has been bothering me for some time. If a user adds mixing, cross mixing, or ventilation to this zone then exhaust might be balanced and we still tell them there is a problem.

Reply:
No, this check here is strictly inside the ductwork. It's basically a mass balance on the mixing box after all the pieces parts have finished. Is supply > return + OA?  The other check at the zone level accounts for the other flows.